source/views/Extension/Views/ImportWallet/components/DragDropBox.jsx

```diff
@@ -0,0 +1,141 @@
+      setFileNotSupported(true);
+      console.log('extension incorrecta', fileExt);
+    }
+  }
+
+  const handleDrag = (e) => {
+    console.log('entra en el handle Drag');
+    e.preventDefault();
+    e.stopPropagation();
+    if (e.type === "dragenter" || e.type === "dragover") {
+      setDragActive(true);
+    } else if (e.type === "dragleave") {
+      setDragActive(false);
+    }
+  }
+
+  const handleDrop = (e) => {
+    console.log('entra en el handle Drop');
+    e.preventDefault();
+    e.stopPropagation();
+    setDragActive(false);
+    if (e.dataTransfer.files && e.dataTransfer.files[0]) {
+      checkDesiredFile(e.dataTransfer.files);
+    }
+  };
+
+  const handleChange = (e) => {
+    console.log('entra en el handleChange');
+    e.preventDefault();
+    e.stopPropagation();
+    if (e.target.files && e.target.files[0]) {
+      checkDesiredFile(e.target.files);
+    }
+  }
+
+
+  return (
+    <>
+      <div className={fileNotSupported ? classes.dragDropContainerError : classes.dragDropContainer}>
+        <form
+          onDragEnter={handleDrag}
+          onDragLeave={handleDrag}
+          onDragOver={handleDrag}
+          onDrop={handleDrop}
+          onSubmit={(e) => e.preventDefault()}
+          className={classes.insideDragDrop}
+        >
+          <input type="file" id='input-file-upload' className={classes.inputFile} multiple={true} onChange={handleChange} />
+          {
+            file ? (
+              <>
+                <img src={GradientFile} alt="gradientFile" />
+                <div className={classes.nameXIcon}>
+                  <Typography variant='h5'>
+                    {file.name || ''}
+                  </Typography>
+                  <CloseIcon className={classes.icon} onClick={() => handleRemoveFile()} />
+                </div>
+              </>
+            ) : (
+              <>
+                {
+                  dragActive ? (
+                    <div className={classes.dropItContainer}>
+                      <Plug size="small" /> 
+                      <span className={classes.dropItLabel}>
+                        {t('importPem.dropIt')}
+                      </span>
+                    </div>
+                  ) : (
+                    <>
+                      <CloudUpload style={{ color: '#BBBEC2' }} fontSize='large' />
+                      <Typography className={classes.dragDropText} variant='h6'>
+                        {t('importPem.dragAndDrop')}<br />
+                        {t('importPem.or')} <label className={classes.inputFileLabel} id="label-file-upload" htmlFor="input-file-upload">{t('importPem.browse')}</label>
+                      </Typography>
+                    </>
+                  )
+                }
+              </>
+            )
+          }
+        </form>
+        {
+          fileNotSupported && (
+            <span className={classes.error}>
+              <img src={RedWarningIcon} alt="" /> {t('importPem.fileNotSupported')}
```

Comment:
why embed the img inside the span? 
it'd be better imo to have a div surrounding an img and a span